### ISSUE TITLE
运行报错

### ISSUE THREAD

**Issue by lonelysheeep** [XGenerationLab/XiYan-DateResolver#3]
# -*- coding: utf-8 -*-
"""
@Author  : shuaixu10
@Tine    : 2025/3/21  15:36
@File    : test_v1.PY
"""
# -*- coding: utf-8 -*-
"""
@Author  : shuaixu10
@Time    : 2025/3/21  11:32
@File    : test.py
"""

import torch
from modelscope import AutoModelForCausalLM, AutoTokenizer
from date_convert import DateTimeUtil  # 确认 date_convert 模块在你的路径中

# 判断设备是否支持 CUDA，如果不支持就使用 CPU
#device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
device = "cuda"
# 指定模型路径
model_name = "/iflytek/yghu3/shuaixu10/model/xiyancode"

# 加载模型和 tokenizer
model = AutoModelForCausalLM.from_pretrained(
    model_name,
    torch_dtype="auto",
    device_map="auto"
)
tokenizer = AutoTokenizer.from_pretrained(model_name)

# 用户问题与 prompt
question = "上周三和上周五的销量是多少？"
prompt = "给你一个用户的问题，请你提取出该用户所提问的时间，结果以list格式输出。\n\n【用户问题】\n{question}\n\n【回答】\n".format(question=question)
messages = [
    {"role": "system", "content": "You are a helpful assistant."},
    {"role": "user", "content": prompt}
]
text = tokenizer.apply_chat_template(
    messages,
    tokenize=False,
    add_generation_prompt=True
)
model_inputs = tokenizer([text], return_tensors="pt").to(device)

generated_ids = model.generate(
    model_inputs.input_ids,
    max_new_tokens=512
)

# 去除输入部分，保留生成的结果
generated_ids = [
    output_ids[len(input_ids):] for input_ids, output_ids in zip(model_inputs["input_ids"], generated_ids)
]

# 解码生成结果
response = tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
list_output = eval(response)
# 将结果转为 Python 对象
# try:
#     list_output = eval(response)
# except Exception as e:
#     print(f"错误：无法解析生成的响应为列表。响应内容：{response}")
#     list_output = []

# 处理时间数据
DateTimeUtil.run(list_output)

2025-03-26 16:23:10,621 - modelscope - INFO - PyTorch version 2.3.0 Found.
2025-03-26 16:23:10,621 - modelscope - INFO - Loading ast index from /root/.cache/modelscope/ast_indexer
2025-03-26 16:23:10,708 - modelscope - INFO - No valid ast index found from /root/.cache/modelscope/ast_indexer, generating ast index from prebuilt!
2025-03-26 16:23:10,760 - modelscope - INFO - Loading done! Current index file version is 1.15.0, with md5 a960b688dff2add07fd781a284eae827 and a total number of 980 components indexed
Loading checkpoint shards: 100%|██████████████████████████████████████████████████| 4/4 [00:08<00:00,  2.21s/it]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
请问这是什么问题
